examples/rf.py

```diff
@@ -100,11 +87,11 @@ def rfr(cfg, seed):
         max_features, min_samples_to_split, min_samples_in_leaf, max_leaf_nodes])
 
 # SMAC scenario oject
-scenario = Scenario({"run_obj": "quality",  # we optimize quality (alternative runtime)
-                     "runcount-limit": 20,  # maximum number of function evaluations
-                     "cs": cs,              # configuration space
-                     "deterministic": "true",
-                     "memory_limit": 1024,
+scenario = Scenario({"run_obj": "quality",   # we optimize quality (alternative runtime)
+                     "runcount-limit": 200,  # maximum number of function evaluations
+                     "cs": cs,               # configuration space
+                     "deterministic": "false",
```

Comment:
If setting this to be random, you should also set a value for `maxR`. In order to not confuse ML people, we should maybe keep this to be deterministic.